### ISSUE TITLE
Talk Ios App  does not work anymore

### ISSUE THREAD

**Issue by Ludovicis** [ramezrafla/spreed-bigbluebutton#8]
## Steps to reproduce
1. Test Talk Ios App with BigBlueButton activate

### Expected behaviour
Talk must work, because without bbb integration the application work as expected.
I think than it's because BBB is not available on mobile app; just on safari on ios.
I don't know on android.
Is it possible to deactivate bbb integration for mobile calls on talk ios app ?

### Actual behaviour
No call no tchat no webcam

**Microphone available:** yes

**Camera available:** yes

**Operating system:** Ios

**Nextcloud 18.03

**Comment by ramezrafla** [comment]
@Ludovicis 

That's right, the native Talk apps will not work as this an HTML5-only client. I should mention this in the README.

BigBlueButton client is designed for HTML5 -- and works in mobile browsers. In fact, we tested it and it works well (we made some optimisations for mobile UI).

I don't think there is a way out. Our plan is to develop a Cordova (i.e. hybrid app) that would load our nextcloud instance in a webview and cache it using ServiceWorkers.

I'll leave this issue open until I update the README
